lib/infrastructure/utils.js

```diff
@@ -56,9 +56,10 @@ module.exports = function Utils() {
 
     function _exec(command, cwd) {
         return new Promise(function(resolve, reject) {
-            exec(command, {cwd: cwd}, function(error, stdout) {
+            exec(command, {cwd: cwd}, function(error, stdout, stderr) {
                 if(error) {
-                    logger.log('Error during "{0}" in "{1}". Output was: {2}'.format(command, cwd, stdout));
+                    logger.log('Error during "{0}" in "{1}".');
```

Comment:
The formatting is missing here and the stdout is desirable too not just the stderr :)